src/nvim/screen.c

```diff
@@ -4328,15 +4388,14 @@ static int win_line(win_T *wp, linenr_T lnum, int startrow, int endrow,
             || filler_todo > 0
             || (wp->w_p_list && wp->w_p_lcs_chars.eol != NUL
                 && p_extra != at_end_str)
-            || (n_extra != 0 && (c_extra != NUL || *p_extra != NUL)))
-        ) {
+            || (n_extra != 0 && (c_extra != NUL || *p_extra != NUL)))) {
       bool wrap = wp->w_p_wrap       // Wrapping enabled.
-        && filler_todo <= 0          // Not drawing diff filler lines.
-        && lcs_eol_one != -1         // Haven't printed the lcs_eol character.
-        && row != endrow - 1         // Not the last line being displayed.
-        && (grid->Columns == Columns  // Window spans the width of the screen,
-            || ui_has(kUIMultigrid))  // or has dedicated grid.
-        && !wp->w_p_rl;              // Not right-to-left.
+                  && filler_todo <= 0 // Not drawing diff filler lines.
+                  && lcs_eol_one != -1 // Haven't printed the lcs_eol character.
+                  && row != endrow - 1 // Not the last line being displayed.
```

Comment:
is it not possible to have 2 spaces before the `//` ?

Reply:
Not sure, I'm still investigating it. Will return with a definitive answer.

Reply:
Fixed :).

It's currently set to have *at least* two spaces before but it's also possible to force it to be exactly two spaces after if that's more preferable.